docker-ubuntu.dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use Ubuntu as the base image
```

Comment:
what's this dockerfile for?

Reply:
Haha both this and the executable snuck in